maxim/logger/portkey/portkey.py

```
@@ -0,0 +1,202 @@
+"""
+This module is a wrapper around the Portkey client that allows for easy integration with Maxim.
+
+It instruments the Portkey client to log to Maxim.
+
+It also provides a wrapper around the Portkey chat completions client that allows for easy integration with Maxim.
```

Pylint (3.3.7)

[convention] Line too long (114/100)

(C0301)